clearwater-socket-factory/clearwater_socket_factory.cpp

```diff
@@ -58,7 +68,96 @@ void logerrno(char* format, ...)
 }
 
 
-int get_shared_socket(char* target)
+struct options
+{
+  std::string signaling_ns;
+  std::vector<std::string> mgmt_allowed_hosts;
+  std::vector<std::string> signaling_allowed_hosts;
+};
+
+
+enum OptionTokens
+{
+  OPT_SIGNALING_NS = 256,
+  OPT_MGMT_ALLOWED_HOSTS,
+  OPT_SIGNALING_ALLOWED_HOSTS
+};
+
+
+const static struct option long_opt[] =
+{
+  {"signaling-ns",            required_argument, NULL, OPT_SIGNALING_NS},
+  {"mgmt-allowed-hosts",      required_argument, NULL, OPT_MGMT_ALLOWED_HOSTS},
```

Comment:
Write command line args in full ("management-allowed-hosts" and "signaling-namespace")